roles/internal/webserver-app/tasks/drupal.yml

```diff
@@ -1,6 +1,6 @@
 ---
 
-- name: Trusted Host Settings
+- name: Trusted Host Settings and more
```

Comment:
Maybe we can rename this to settings.php settings or something generic?

Reply:
There's more than just this singular task doing `settings.php` things... really, was just trying to reflect the reality in the name that it was doing more than just the "trusted host" stuff it was originally named... like: Like the flysystem and `$content_directories` stuff... and trying to follow something of the established patterns of naming things.